From 23a13cde0d81a8776ab4defcd30f9edef53b045f Mon Sep 17 00:00:00 2001
Message-Id: <23a13cde0d81a8776ab4defcd30f9edef53b045f.1346843178.git.minovotn@redhat.com>
From: Michal Novotny <minovotn@redhat.com>
Date: Wed, 18 Jul 2012 14:03:17 +0200
Subject: [PATCH 1/5] qemu-options.hx: Improve -nodefaults description

RH-Author: Michal Novotny <minovotn@redhat.com>
Message-id: <87fc301f92ed5d2a8b58aafd07bb3e587f8f3dd2.1342619852.git.minovotn@redhat.com>
Patchwork-id: 40372
O-Subject: [RHEL-6.4 qemu-kvm PATCH v2] qemu-options.hx: Improve -nodefaults description
Bugzilla: 817224
RH-Acked-by: Amos Kong <akong@redhat.com>
RH-Acked-by: Markus Armbruster <armbru@redhat.com>
RH-Acked-by: Pavel Hrdina <phrdina@redhat.com>

Bugzilla: 817224
Upstream commit: 66c19bf1204f2cb283918c24d4eb74d02c5e882c

This patch improves the description of -nodefaults QEMU command line
option by adding more information what is being disabled using this
command.

Signed-off-by: Michal Novotny <minovotn@redhat.com>
Signed-off-by: Anthony Liguori <aliguori@us.ibm.com>
---
 qemu-options.hx |    4 ++++
 1 file changed, 4 insertions(+)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 qemu-options.hx | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/qemu-options.hx b/qemu-options.hx
index 7209512..1b20237 100644
--- a/qemu-options.hx
+++ b/qemu-options.hx
@@ -2101,6 +2101,10 @@ ETEXI
 DEF("nodefaults", 0, QEMU_OPTION_nodefaults, \
     "-nodefaults     don't create default devices.\n")
 STEXI
+Don't create default devices. Normally, QEMU sets the default devices like serial
+port, parallel port, virtual console, monitor device, VGA adapter, floppy and
+CD-ROM drive and others. The @code{-nodefaults} option will disable all those
+default devices.
 ETEXI
 
 #ifndef _WIN32
-- 
1.7.11.4

